src/frontend/packages/lib_classroom/src/components/ClassroomWidgetProvider/widgets/Recordings/Recording/index.tsx

```diff
@@ -0,0 +1,212 @@
+                  title={intl.formatMessage(messages.vodDashboardLabel)}
+                >
+                  <Text color="blue-active" size="0.9rem">
+                    {recording.vod.title}
+                  </Text>
+                </NavLinkStyled>
+              )}
+              {ltiLink && (
+                <CopyClipboard
+                  copyId={`ltiLink-${recording.vod.id}`}
+                  text={ltiLink}
+                  title={intl.formatMessage(messages.ltiLinkLabel)}
+                  withLabel={true}
+                  onSuccess={() => {
+                    toast(intl.formatMessage(messages.ltiLinkCopiedSuccess), {
+                      icon: '📋',
+                    });
+                  }}
+                  onError={(event) => {
+                    toast.error(event.text, {
+                      position: 'bottom-center',
+                    });
+                  }}
+                />
+              )}
+            </React.Fragment>
+          ) : (
+            <Box
+              direction="row"
+              key={recording.id}
+              align="center"
+              fill="horizontal"
+              gap="medium"
+              pad="none"
+              justify="between"
+            >
+              <StyledAnchor
+                title={intl.formatMessage(messages.downloadRecordingLabel)}
+                href={recording.video_file_url}
+                target="_blank"
+                rel="noreferrer noopener"
+              >
+                <Text color="blue-active" size="0.9rem" truncate>
+                  {recording.vod.title}
+                </Text>
+              </StyledAnchor>
+              <Text color="blue-active" size="0.9rem">
+                {recording.vod.upload_state}
+              </Text>
+            </Box>
+          )}
+        </Box>
+      ) : (
+        <Box
+          direction="row"
+          key={recording.id}
+          align="center"
+          fill="horizontal"
+          gap="medium"
+          pad="small"
+          justify="between"
+        >
+          <StyledAnchor
+            title={intl.formatMessage(messages.downloadRecordingLabel)}
+            href={recording.video_file_url}
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            <Text color="blue-active" size="0.9rem" truncate>
+              {buildRecordingTitle(recording, intl)}
+            </Text>
+          </StyledAnchor>
+          <Button
+            a11yTitle={intl.formatMessage(messages.convertVODTitle, {
+              recordingTitle: buildRecordingTitle(recording, intl),
+            })}
+            title={intl.formatMessage(messages.convertVODTitle, {
+              recordingTitle: buildRecordingTitle(recording, intl),
+            })}
+            label={intl.formatMessage(messages.convertVODLabel)}
+            onClick={() => convertVOD(recording)}
+            size="xsmall"
+            pad="xsmall"
+          />
+        </Box>
```

Comment:
This 2 blocks are very similar, what about creating 1 small component, it could increase readability IMO as well?